src/main/java/com/xeeva/catalog/CostCenterPage.java

```diff
@@ -152,16 +267,172 @@ public void change_CC(String changeType,String CC){
 			String itemNumber = driver.findElement(By.xpath(".//*[@id='trCCRow_"+randomID+"']/td/a")).getText();
 			TestReporter.logStep("itemNumber :"+ itemNumber);
 			ddCCLineLevel.get(0).select(CC);
-			TestReporter.assertTrue(verifyCC_LineLevel(itemNumber,CC),"Cost Center Updated at Line Level Successfully!!");
+			System.out.println(verifyCostCenter(changeType,itemNumber,CC,null));
+			TestReporter.assertTrue(verifyCostCenter(changeType,itemNumber,CC,null),"Cost Center Updated at Line Level Successfully!!");
 			break;
 
 		case "headerlevel" :
 			lstCostCenterHeaderLevel.select(CC);
+			//Handle Alert if present
+			if(AlertHandler.isAlertPresent(driver, 6)){
+				AlertHandler.handleAlert(driver, 6);
+			}
+			System.out.println(verifyCostCenter(changeType,null,CC,null));
+			TestReporter.assertTrue( verifyCostCenter(changeType,null,CC,null),"Cost Center Updated at Header Level Successfully!!");
 			break;
 
 		default : System.out.println();
 		}
 	}
-}
 
+	public String selectCCToCopyItem(){
+		driver.setPageTimeout(4);

```

Comment:
There is no need to change the Page timeout here.  All the page time out does is change the time it takes for a page to load when using the commands driver.get(URL) or driver.navigate(URL).  Please remove